packages/react-native-aztec/src/AztecView.js

```diff
@@ -234,6 +235,22 @@ class AztecView extends Component {
 		return focusedElement && focusedElement === this.aztecViewRef.current;
 	}
 
+	onRemoveMarkFormatting() {
+		UIManager.dispatchViewManagerCommand(
+			findNodeHandle( this.aztecViewRef.current ),
+			'onRemoveMarkFormatting',
+			[]
+		);
+	}
+
+	onMarkFormatting( color ) {
+		UIManager.dispatchViewManagerCommand(
+			findNodeHandle( this.aztecViewRef.current ),
+			'onMarkFormatting',
+			[ color ]
+		);
```

Comment:
Similar to my previous comment, I wonder if could use the function `dispatch` provided in this class.
```suggestion
		this.dispatch( AztecManager.Commands.onMarkFormatting, [ color ] );
```

Reply:
Same as [this comment](https://github.com/WordPress/gutenberg/pull/57650/files#r1516487893)